homeassistant/components/lektrico/binary_sensor.py

```diff
@@ -0,0 +1,118 @@
+"""Support for Lektrico charging station binary sensors."""
+
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Any
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    ATTR_IDENTIFIERS,
+    ATTR_MANUFACTURER,
+    ATTR_MODEL,
+    ATTR_NAME,
+    ATTR_SW_VERSION,
+    CONF_FRIENDLY_NAME,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import LektricoDeviceDataUpdateCoordinator
+from .const import DOMAIN
+
+
+@dataclass
+class LektricoBinarySensorEntityDescription(BinarySensorEntityDescription):
+    """A class that describes the Lektrico binary sensor entities."""
+
+    @classmethod
+    def get_is_on(cls, data: Any) -> bool | None:
+        """Return None."""
+        return None
+
+
+@dataclass
+class HeadlessBinarySensorEntityDescription(LektricoBinarySensorEntityDescription):
+    """A class that describes the Lektrico No Authentication binary sensor entity."""
+
+    @classmethod
+    def get_is_on(cls, data: Any) -> bool:
+        """Get the headless."""
+        return bool(data.headless)
```

Comment:
This class/method isn't needed when using a property in the entity described above.
The entity description definition below can, in that case, pass a lambda to get the data and return the result

Reply:
I changed it, thanks.